tests/test_imaging.py

```diff
@@ -1,9 +1,16 @@
 import shutil
+import sys
 import tempfile
 
+import pytest
+
 from constructor.imaging import write_images
```

Comment:
Wouldn't this trigger an import error for pillow? We might want to guard this import.

Reply:
You are correct, I added a guard.